docs/.vuepress/public/presets/toml/pastel-powerline.toml

```
@@ -117,6 +118,11 @@ symbol = ""
 style = "bg:#86BBD8"
 format = '[ $symbol ($version) ]($style)'
 
+[scala]
+symbol = ""
```

Most of the symbols in the preset have a space after them (though I do wonder if the double-space is intentional):
```suggestion
symbol = " "
```